### PR TITLE
Integrated noise Gaussian uniform likelihood (rebased)

### PR DESCRIPTION
**Pull request by ben18785**
- Rebased after substantial changes to master
- Adapted text as per @MichaelClerx 's comments on #731

### CODE REVIEW

**Comment by ben18785** [comment]
@MichaelClerx Have done my best to rebase this after the various changes to the master branch...sorry if I've missed anything!

File: pints/tests/test_log_likelihoods.py [hunk 1]
```diff
@@ -559,30 +538,6 @@ def test_ar1(self):
         self.assertAlmostEqual(
             log_likelihood([0, 0.5, 5]), -19.706737485492436)
 
-        # multiple outputs
```
Comment (MichaelClerx) [inline pints/tests/test_log_likelihoods.py]: Hiya! What's hapening here and below?

Comment (ben18785) [reply pints/tests/test_log_likelihoods.py]: Cool, sorry -- I didn't do the rebasing properly. See #752

**Comment by ben18785** [comment]
Replaced by #752

**Pull request closed without merging**